src/components/organisms/EntityDetails/EntityDetailsContent/Settings/SettingsGeneral/SettingsGeneral.tsx

```diff
@@ -17,6 +18,7 @@ const SettingsGeneral: React.FC = () => {
       <NameNDescription />
       <Labels />
       {entity === 'tests' ? <Timeout /> : null}
```

Comment:
Do we prefer this convention over equivalent  {entity === 'tests' && <Timeout />}

Reply:
+1